python/lsst/cp/pipe/defects.py

```diff
@@ -716,3 +762,154 @@ def run(self, inputDefects, camera):
+
+    ConfigClass = MergeDefectsCombinedTaskConfig
+    _DefaultName = "cpDefectMergeCombined"
+
+    def runQuantum(self, butlerQC, inputRefs, outputRefs):
+        inputs = butlerQC.get(inputRefs)
+        # Turn inputDefects and secondaryInputDefects into a list
+        # which is what MergeDefectsTask expects.
+        tempList = [inputs['inputDefects'], inputs['secondaryInputDefects']]
+        # Rename inputDefects
+        inputsCombined = {'inputDefects': tempList, 'camera': inputs['camera']}
+
+        outputs = super().run(**inputsCombined)
+        butlerQC.put(outputs, outputRefs)
+
+
+class MeasureDefectsCombinedConnections(MeasureDefectsConnections,
+                                        dimensions=("instrument", "detector")):
+    inputExp = cT.Input(
+        name="defectExps",
+        doc="Input ISR-processed combined exposures to measure.",
+        storageClass="Exposure",
+        dimensions=("instrument", "detector"),
+        multiple=False,
+        isCalibration=True,
+    )
+    camera = cT.PrerequisiteInput(
+        name='camera',
+        doc="Camera associated with this exposure.",
+        storageClass="Camera",
+        dimensions=("instrument", ),
+        isCalibration=True,
+        lookupFunction=lookupStaticCalibration,
+    )
+
+    outputDefects = cT.Output(
+        name="combinedExpDefects",
+        doc="Output measured defects.",
+        storageClass="Defects",
+        dimensions=("instrument", "detector"),
+    )
+
+
+class MeasureDefectsCombinedTaskConfig(MeasureDefectsTaskConfig,
+                                       pipelineConnections=MeasureDefectsCombinedConnections):
+    """Configuration for measuring defects from combined exposures.
+    """
+    pass
+
+
+class MeasureDefectsCombinedTask(MeasureDefectsTask):
+    """Task to measure defects in combined images."""
+
+    ConfigClass = MeasureDefectsCombinedTaskConfig
+    _DefaultName = "cpDefectMeasureCombined"
+
+
+class MeasureDefectsCombinedWithFilterConnections(MeasureDefectsCombinedConnections,
+                                                  dimensions=("instrument", "detector")):
+    """Task to measure defects in combined flats under a certain filter."""
+    inputExp = cT.Input(
+        name="defectExps",
+        doc="Input ISR-processed combined exposures to measure.",
+        storageClass="Exposure",
+        dimensions=("instrument", "detector", "physical_filter"),
+        multiple=False,
+        isCalibration=True,
+    )
+    camera = cT.PrerequisiteInput(
+        name='camera',
+        doc="Camera associated with this exposure.",
+        storageClass="Camera",
+        dimensions=("instrument", ),
+        isCalibration=True,
+        lookupFunction=lookupStaticCalibration,
+    )
+
+    outputDefects = cT.Output(
+        name="combinedExpDefects",
+        doc="Output measured defects.",
+        storageClass="Defects",
+        dimensions=("instrument", "detector"),
```

Comment:
Doesn't this need the `physical_filter` dimension, or else you could clobber yourself if you ran multiple filters of data through the defect pipeline?

Reply:
This name also needs updating.

Reply:
Changed the name to `cpPartialDefectsFromFlatCombined`. What you say about the filter dimension makes sense to me. @czwa , what are your thoughts?

Reply:
I hadn't thought of the case where multiple flats could be found.  Keeping the filter dimension makes sense to me now as well.  The associated connection in the merge code needs to be updated as well to match dimensions, but that task can still have dimensions of `(instrument, detector)`.